src/devices/cpu/drcbec.cpp

```diff
@@ -1579,6 +1619,15 @@ int drcbe_c::execute(code_handle &entry)
 				flags = dmulu(*inst[0].puint64, *inst[1].puint64, DPARAM2, DPARAM3, true);
 				break;
 
+			case MAKE_OPCODE_SHORT(OP_MULULW, 8, 0):      // DMULULW  dst,src1,src2[,f]
+				dmulu(*inst[0].puint64, *inst[0].puint64, DPARAM1, DPARAM2, false);
+				break;
+
+			case MAKE_OPCODE_SHORT(OP_MULULW, 8, 1):
+				flags = dmulu(*inst[0].puint64, *inst[0].puint64, DPARAM1, DPARAM2, true);
+				flags = FLAGS64_NZ(DPARAM0) | (flags & FLAG_V);
+				break;
```

Comment:
This would appear to be inconsistent with the 32-bit version in that it doesn’t set V if the result is truncated.

Reply:
FLAG_V is the overflow flag that I mentioned before, where it sets overflow/FLAG_V if the upper half of the result isn't zero. `dmulu` sets FLAG_V using `((hi != 0) << 1)` so I think it's the same.

Can you think of a specific case where you think it'd fail?

Reply:
Oh sorry, I forgot `dmulu` was already a DRC thing that set flags and not just a multiplication.